solidity/contracts/lib/zeto_base.sol

```diff
@@ -118,4 +190,67 @@ abstract contract ZetoBase is IZetoBase, ZetoCommon {
         }
         emit UTXOMint(utxos, msg.sender, data);
     }
+    // Locks the UTXOs so that they can only be spent by submitting the appropriate
+    // proof from the Eth account designated as the "delegate". This function
+    // should be called by escrow contracts that will use uploaded proofs
```

Comment:
Small note, but "lock" wouldn't actually be called by the escrow contract - it would have to be called by the account owner establishing the the escrow contract.